Makefile

```diff
@@ -65,7 +65,12 @@ release-static-arm6:
 release-static-arm7:
 	mkdir -p build/release
 	cd build/release && cmake -D BUILD_TESTS=OFF -D ARCH="armv7-a" -D STATIC=ON -D BUILD_64=OFF -D CMAKE_BUILD_TYPE=release ../.. && $(MAKE)
-	
+
+release-static-armv8:
+	#May need to account for armv8.1 and armv8.2 in future
+	mkdir -p build/release
+	cd build/release && cmake -D BUILD_TESTS=OFF -D ARCH="armv8-a" -D STATIC=ON -D BUILD_64=ON -D CMAKE_BUILD_TYPE=release ../.. && $(MAKE)
```

Comment:
Yeah, this stuff will never be simple, complexity will only increase with more hardware each with its own quirks.